strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/io/RepositoryFileSystemProvider.java

```diff
@@ -42,20 +42,31 @@
  *
  * @author Sergey Bespalov
  */
-public class RepositoryFileSystemProvider
+public abstract class RepositoryFileSystemProvider
         extends FileSystemProvider
 {
 
     private static final Logger logger = LoggerFactory.getLogger(RepositoryFileSystemProvider.class);
 
     private FileSystemProvider storageFileSystemProvider;
-
+    private boolean allowsForceDelete;
+    
     public RepositoryFileSystemProvider(FileSystemProvider storageFileSystemProvider)
     {
         super();
         this.storageFileSystemProvider = storageFileSystemProvider;
     }
 
+    public boolean isAllowsForceDelete()
```

Comment:
are you sure about that? this is common naming convention for `boolean` `getters/setters`.